### DISCUSSION TITLE
FFCM1_Red citation

### DISCUSSION THREAD

**Discussion by RSuryaNarayan** [AMReX-Combustion/PelePhysics#484] [Q&A]
Hello

Is there a citation for the mechanism in ```Mechanisms/FFCM1_Red``` ? The actual FFCM1 looks to be larger than the one here..

**Comment by baperry2** [comment]
@esclapez do you know where this reduction comes from? This is the initial commit for the mechanism: https://github.com/AMReX-Combustion/PelePhysics/commit/62b0a8b9a2ff2a539c478f2b6df8c6bf83740971

**Reply by EnnaDelfen** [reply]
Hey guys, I just sent an email to Surya. While I submitted the original commit, I can't remember where this mechanism comes from. My guess is that it was somehow a part of Fuego and I simply reorganized/renamed the files. 
I thought that the mechanism was the FFCM1-21 skeletal reduced mechanism cited in [this paper ](https://doi.org/10.1016/j.combustflame.2024.113380), developed by Tianfeng Lu based on the original [detailed FFCM version 1 mechanism ](https://web.stanford.edu/group/haiwanglab/FFCM1/pages/FFCM1.html). But after comparing the mechanisms, it turns out that they do not match. So I am at a loss. I will research a bit more, maybe @drummerdoc would be able to enlighten us ?

**Reply by drummerdoc** [reply]
I had no enlightenment to share, so I reached out to Hai directly.  I will upload a newer reduced mech that he supports, and will likely remove the mystery one.

**Reply by baperry2** [reply]
Thanks. I support removing anything mysterious. It's part of #460 to track down the sourcing for all mechanisms shipped with PelePhysics and add appropriate citations to the documentation, and potentially remove some if no longer needed, but I have no concrete plans on when that will happen.

**Reply by EnnaDelfen** [reply]
This all sounds reasonable. Hai's version is probably just as compact so perhaps best to switch to it if it's not too late, Surya ...

**Reply by RSuryaNarayan** [reply]
@EnnaDelfen @drummerdoc  Thanks a lot for the information. I will try the FFCM1-21 mechanism to see if it behaves similar to ```FFCM1_Red``` inside Pele and keep you posted! If it works, as @baperry2 mentioned, we can remove the old one.
